### DISCUSSION TITLE
Question 74: Taxonomy records created [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#652] [Q&A]
Description: I made 4 new taxonomy records and wanted to make sure I did this correctly. I was a little unsure about how to add authority. IRNs: 763297, 763298, 763299, 763300
I also tried to make Lonchophylla thomasi a synonym of Hsunycteris thomasi by following [this workflow](https://docs.google.com/document/d/1PnftDxe-oRp1VFrlQsoFSFx796S4xKzPJxdLJDtdSv0/edit) but got this error message below:  

![unnamed](https://user-images.githubusercontent.com/702709/187553084-bdb8b83e-30a7-4acd-af3d-b18db1d966fa.png)

I do not have edit privileges to change L. thomasi so I could not change “Is Original Name?” field on Primary Citation tab to “Yes”. This may be the issue.

Url example: https://www.mammaldiversity.org/explore.html#genus=Hsunycteris&species=thomasi&id=1004927

**Comment by rondlg** [comment]
Response: Do you have authors and years? If so, we can walk you through how to decide where to put them. Also, is there a reason why irn 763299 has only “Action” in the Security Department field or was it an oversight?
	-Yes, it was an oversight and it is fixed now.

Authority should go on Authors tab.

Github issue will have a fuller description on how to handle the url (&/or id at end of url) either as a bibliography attachment and/or in the GUIDs field on the Admin tab.

AF to request/add ZK to a Mammals user group
This link should answer questions about both Synonyms and Authors https://github.com/fieldmuseum/EMu-Documentation/discussions/498

**Reply by zkachian** [reply]
I looked over these docs linked on github and it had info about how to create synonymies, but I didn't see anything about how to add authors.

**Reply by fmjjones** [reply]
Try this link: [Axiell Help](https://help.emu.axiell.com/v5.1/en/Topics/EMu/Citations%20tab%20-%20Taxonomy%20module.htm) and read thru the Citation, Primary Citation, and Authors Tab.

**Reply by zkachian** [reply]
I read through the Axiell help doc but I'm still not sure how I should be doing this exactly. It looks like the Axiell doc assumes you'll be adding a bibliography record on the citations tab, but how should I be doing this if I don't have a citation (only Author and Year)? Record 763298 used the Parenthetic Authors field on the Authors tab, but the Axiell doc doesn't mention the Parenthetic Authors field. Is this correct? I'm also not sure how Parenthetic Authors is different from Name Authors.

**Reply by fmjjones** [reply]
Try this https://github.com/fieldmuseum/EMu-Documentation/discussions/660

**Reply by zkachian** [reply]
That helped, thank you! I think that IRNs: 763297, 763298, 763299, 763300 are all correct now

**Comment by rondlg** [comment]
_From Adam Ferguson_ 
Hello Zach, thanks for the email heads up on the new names, all of them check out as currently accepted names so that's good. I think the one thing I might suggest is if it's possible to include the hyperlink to the individual taxon via the Mammal Diversity Database. E.g., for Tonatia maresi [https://www.mammaldiversity.org/explore.html#genus=Tonatia&species=maresi&id=1004986](https://www.google.com/url?q=https://www.mammaldiversity.org/explore.html%23genus%3DTonatia%26species%3Dmaresi%26id%3D1004986&sa=D&source=docs&ust=1661897210216957&usg=AOvVaw0B4ax1q8DmmSYzZLivNtcT)

_From Zach Kachian_
Hi Adam, I was looking at the updates you made to 763298 and 312608 for creating a synonym. It looks like you added the taxa on the Homotypics tab, but the workflow I linked says to add the taxa on the Primary Citation tab. Although everything looks correct on Classification tab now. But I just wanted to check which way I should be doing this in the future? Thanks
